environment/Dockerfile

```diff
@@ -29,23 +30,26 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   xz-utils
 
 # Install aptly for managing mirror debs
-RUN echo "deb http://repo.aptly.info/ squeeze main" >> /etc/apt/sources.list && \
-    curl --silent https://www.aptly.info/pubkey.txt | apt-key add - && \
-    apt-get update && apt-get install --no-install-recommends -y aptly
+ADD https://github.com/aptly-dev/aptly/releases/download/v1.5.0/aptly_1.5.0_linux_amd64.tar.gz /tmp
```

Comment:
perhaps is better to use something like `curl` or `wget` to get this file instead of `ADD`? I think `ADD` does not give you feedback of progress of download and things like this

Reply:
good idea, I've also merged this and the untar into a single line